pkg/data/components/OTelGRPCExporter.yaml

```diff
@@ -24,13 +24,22 @@ ports:
     direction: input
     type: OTelLogs
 properties:
+  - name: Headers
+    summary: Headers to emit when sending gRPC traffic.
+    description: |
+      Sending data to a backend may require additional headers to be
+      configured. This property supports sending a map of header keys and
+      values.
+    type: map
+    validations: []
```

Comment:
Oh good idea, assume this was moved to the top bc it's not advanced, and the advanced properties will be separated below it.